tests/test_lime_survey.py

```diff
@@ -1,17 +1,20 @@
 """Test functions related to Survey class"""
 import os
+import re
 import unittest
 
 import pandas as pd
 from pandas._testing import assert_frame_equal
 
 from n2survey.lime import LimeSurvey, read_lime_questionnaire_structure
 
+structure_file = "data/survey_structure_2021.xml"
+response_file = "data/dummy_data_2021_codeonly.csv"
```

Comment:
As a best practice, global variable names are in `CAPTAL_SNAKE_CASE`, i.e. `STRUCTURE_FILE` and `RESPONSE_FILE`.
Sorry, did not notice it on my first review because I was focusing on the main function more.